torcheeg/transforms/numpy/downsample.py

```diff
@@ -60,4 +61,53 @@ def repr_body(self) -> Dict:
         return dict(super().repr_body, **{
             'num_points': self.num_points,
             'axis': self.axis
-        })
\ No newline at end of file
+        })
+    
+
+
+
+
+class SetSamplingRate(EEGTransform):
+    r'''
+    Change the EEG signal to another sampling rate.
+
+    .. code-block:: python
+
+        from torcheeg import transforms
+
+        t = SetSamplingRate(origin=500,target_sampling_rate=128)
+        t(eeg=np.random.randn(32, 1000))['eeg'].shape
+        >>> (32, 256)
+
+    Args:
+        origin (int): Original sampling rate of EEG.
+        target_sampling_rate (int): Target sampling rate of EEG.
+        apply_to_baseline: (bool): Whether to act on the baseline signal at the same time, if the baseline is passed in when calling. (default: :obj:`False`)
+    
+    .. automethod:: __call__
+    '''
+    def __init__(self,origin:int, target_sampling_rate:int, apply_to_baseline=False):
+        super(SetSamplingRate, self).__init__(apply_to_baseline=apply_to_baseline)
+        self.original_rate = origin
+        self.new_rate = target_sampling_rate
+
+
+    def apply(self, eeg, **kwargs) -> any:
+        assert len(eeg.shape) == 2, 'Make sure the EEG in 2D shape.'
```

Comment:
The EEG signals to be downsampled do not have to be 2D. A typical example is time series signals of different frequency bands represented by [frequency band, electrode, data point].